packages/synchro-charts/src/components/sc-widget-grid/sc-grid-tooltip.tsx

```diff
@@ -61,12 +70,13 @@ export class ScGridTooltip {
               <div class="awsui-util-spacing-v-s">
                 {this.propertyPoint && (
                   <div>
-                    <div class="awsui-util-label">Latest value:</div>
+                    <div class="awsui-util-label">{this.messageOverrides?.tooltipValueTitles || 'Latest value:'}</div>
```

Comment:
please refer to https://github.com/awslabs/iot-app-kit/pull/211/files on how to handle message overrides.

we dont want the default values hard coded everywhere, instead they should be defined in one spot, and merged together with what the user passes in.

Reply:
Done